packages/chart/src/components/SparkLine.js

```diff
@@ -90,10 +90,19 @@ export default function SparkLine({ width: parentWidth, height: parentHeight })
 
   const handleSparkLineTicks = [xScale.domain()[0], xScale.domain()[xScale.domain().length - 1]]
 
+  const isNumber = value => {
```

Comment:
Better naming used in this fn than checkCy in the other file. If they're the same, lift the functions to CdcChart and pass through.

Reply:
yep

Reply:
Moved isNumber to CdcChart ... looking at checkCY... cant recall if we need that or not